samples/LibVLCSharp.Eto.Sample/Program.cs

```diff
@@ -0,0 +1,22 @@
+
+namespace LibVLCSharp.Eto.Sample
+{
+    using System;
+    using global::Eto.Forms;
+
+    public static class Program
+    {
+        [STAThread]
+        public static void Main(string[] args)
+        {
+            try
+            {
+                new Application(global::Eto.Platform.Detect).Run(new MainForm());
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, "Exception !", MessageBoxType.Error);
```

Comment:
MessageBox.Show throws here if the Application was not created yet (like `new Application(global::Eto.Platforms.Gtk2)` when I did not install the corresponding package.